### ISSUE TITLE
Accuracy on LFW, CFP and AgeDB30

### ISSUE THREAD

**Issue by wjgaas** [KaleidoZhouYN/MobileResNet-50#3]
this work looks great!
i test it on the LFW, CFP and AgeDB30 with the model you provided, but got a low accuracy.
i use the lfw.bin, cfp.bin and angedb30.bin in faces_emore, and test as follow:
data = (data - 127.5) * 0.0078125
embeddings = model.forward()['View_1']
embeddings = sklearn.preprocessing.normalize(embeddings)
Then, i got the results:
[lfw]Accuracy: 0.99750+-0.00318
[cfp_fp]Accuracy: 0.95586+-0.00671
[agedb_30]Accuracy: 0.96983+-0.00864
with image flip:
[lfw]Accuracy-Flip: 0.99733+-0.00281
[cfp_fp]Accuracy-Flip: 0.96057+-0.00783
[agedb_30]Accuracy-Flip: 0.97167+-0.00910

Is there anything wrong? or maybe you use images with more accurate aligned?
